docs/contributing.rst

```diff
@@ -7,40 +7,33 @@ Contributing
 Hacking
 =======
 
-All changes in your pull request **must** have 100% unit test coverage, pass
-our `integration`_ tests, **and** be compliant with the
-:ref:`coding style <coding-style>`.
+Running a local copy of the client
+----------------------------------
 
-
-Bootstrap
----------
-
-Start by :ref:`installing Let's Encrypt prerequisites
-<prerequisites>`. Then run:
+Running the client in developer mode from your local tree is very similar to
+running the released versions of the client; just replace ``letsencrypt-auto``
+with ``letsencrypt-dev``:
 
 .. code-block:: shell
 
-   ./bootstrap/dev/venv.sh
+   git clone https://github.com/letsencrypt/letsencrypt
+   cd letsencrypt
+   ./letsencrypt-dev
 
-Activate the virtualenv:
 
-.. code-block:: shell
+Find issues to work on
+----------------------
 
-   source ./$VENV_NAME/bin/activate
+You can find the open issues in the `github issue tracker`_.  If you're
+starting work on something, post a comment to let others know and seek
+feedback on your plan where appropriate.
 
-This step should prepend you prompt with ``($VENV_NAME)`` and save you
```

Comment:
That's because you're using OS-wide `tox` and you shouldn't do that, because it causes strange side-effects that are hard to debug!

In fact most of the people won't even have OS-wide `tox` installed.